I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 40, in script
    papyrus "I walked past a sign saying, \"Let's give it 100%!\" yeah"
ValueError: unsupported format character '!' (0x21) at index 48

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 40, in script
    papyrus "I walked past a sign saying, \"Let's give it 100%!\" yeah"
  File "C:\Program Files (x86)\renpy-6.99.10-sdk\renpy\ast.py", line 604, in execute
    renpy.exports.say(who, what, interact=self.interact)
  File "C:\Program Files (x86)\renpy-6.99.10-sdk\renpy\exports.py", line 1114, in say
    what = what % tag_quoting_dict
ValueError: unsupported format character '!' (0x21) at index 48

Windows-8-6.2.9200
Ren'Py 6.99.10.1227
Undertale Dating Simulator (Demo) 0.0
